wide_deep/torch_model.py

```diff
@@ -191,8 +192,12 @@ def fit(self, dataset, n_epochs, batch_size):
                     X_w, X_d, y = X_w.cuda(), X_d.cuda(), y.cuda()
 
                 self.optimizer.zero_grad()
-                y_pred =  net(X_w, X_d)
-                loss = self.criterion(y_pred, y)
+                y_pred =  net(X_w, X_d) # [batch_size, 1]
+                loss = None
```

Comment:
I don't think we need `loss = None`?

Reply:
@jrzaurin 
This is just because of the following codes.
```
if(self.criterion == F.cross_entropy):
    loss = self.criterion(y_pred, y) #[batch_size, 1]
else:
    loss = self.criterion(y_pred, y.view(-1, 1)) #[batch_size, 1]
```
In the stable version of PyTorch,  the implicit dimension choice for {} has been deprecated when we use `F.softmax `.
https://github.com/pytorch/pytorch/blob/master/torch/nn/functional.py#L1191
So, I think there are two different cases when we run `loss = self.criterion`.

Thank you very much for your review!! :)

Reply:
is merged, but I meant precisely the `loss = None` line. 

There is no need of that line, right. The rest is fine

Reply:
Now I can accurately understand what you mean. :)
You're right. Python doesn't need it.
I appreciate you merge my poor codes nevertheless you knew it.